app/jobs/fastly_log_processor.rb

```diff
@@ -0,0 +1,75 @@
+require 'zlib'
+FastlyLogProcessor = Struct.new(:bucket, :key) do
```

Comment:
I see you've made the change already, but for the curious, Structs are ~46% _slower_ than using a Class (at least, for simple getter/setter actions.) [Some benchmarking](https://gist.github.com/kerrizor/e961504d1fbd425bfe97)